app/menus/DocumentMenu.tsx

```diff
@@ -225,8 +228,9 @@ function DocumentMenu({
       <ContextMenu
         {...menu}
         aria-label={t("Document options")}
-        onOpen={handleOpen}
+        onOpen={onOpen}
         onClose={onClose}
+        loading={loading || documents.isFetching}
```

Comment:
I don't think we want to show a loader here just because subscriptions is fetching, that's going to be quite annoying.

Reply:
So, we don't show the loader at all?

Reply:
Right now sometimes the subscribe will switch to unsubscribe after a couple of ms, that seems a lot less jarring than the entire menu being in a loading state